### DISCUSSION TITLE
Can I "O" across floor levels? So I can "Y" a zone in 2nd floor and order things in the 1th floor.

### DISCUSSION THREAD

**Discussion by linonetwo** [CleverRaven/Cataclysm-DDA#47084] [Q&A] [answered]
I tried this in a wizard tower, and things in the first floor didn't come to the 2nd floor.

**Comment by actual-nh** [comment]
If you are meaning this as a feature request, it might be better as an issue of that type.

**Reply by linonetwo** [reply]
Okay， I thought I could do this in some way special.

**Reply by actual-nh** [reply]
Ah. That, I don't know. It doesn't sound like it - or, if it is possible, the UI needs improvement so it isn't hard to figure out.

**Comment by anothersimulacrum** [comment, marked as answer]
No.
